lisa/sut_orchestrator/libvirt/transformers.py

```diff
@@ -551,7 +551,6 @@ def _install_libvirt(runbook: schema.TypedSchema, node: Node, log: Logger) -> No
 
         if isinstance(node.os, Ubuntu):
             node.execute("systemctl disable apparmor", shell=True, sudo=True)
-        node.execute("systemctl enable libvirtd", shell=True, sudo=True)
         node.reboot(time_out=900)
```

Comment:
Since a reboot is already done at line 577, just after this if-else block, this seems redundant.

Reply:
Nice catch, I will fix this.